File: Codewars/8kyu/8kyu_basic_mathmatical_operations/src/main.py
def basic_op(operator, value1, value2):
    return eval('%d%s%d' % (value1, operator, value2))
    
print(basic_op('+', 4, 7) == 11)
print(basic_op('-', 15, 18) == -3)
print(basic_op('*', 5, 5) == 25)
print(basic_op('/', 49, 7) == 7)

File: Codewars/8kyu/8kyu_basic_mathmatical_operations/src/main.js
function basicOp(operation, value1, value2) {
   return eval(`${value1}${operation}${value2}`)
}

console.log(basicOp('+', 4, 7) == 11)
console.log(basicOp('-', 15, 18) == -3)
console.log(basicOp('*', 5, 5) == 25)
console.log(basicOp('/', 49, 7) == 7)
